.clang-format

```diff
@@ -1,11 +1,28 @@
 ---
-BasedOnStyle: Mozilla
+BasedOnStyle:                           Mozilla
+ColumnLimit:                            100
+IndentWidth:                            4
+ContinuationIndentWidth:                4
+TabWidth:                               4
+UseTab:                                 Never
+AccessModifierOffset:                   -4
+PointerAlignment:                       Middle
+BreakBeforeBraces:                      Custom
+BraceWrapping:
+  AfterClass:                           false
+  AfterControlStatement:                false
+  AfterEnum:                            true
+  AfterFunction:                        true
+  AfterNamespace:                       false
+  AfterObjCDeclaration:                 false
+  AfterStruct:                          true
+  AfterUnion:                           true
+  BeforeCatch:                          false
+  BeforeElse:                           true
+  IndentBraces:                         false
+AlwaysBreakBeforeMultilineStrings:      true
+#AlignAfterOpenBracket:                  AlwaysBreak
 ---
-Language: Cpp
-AllowAllParametersOfDeclarationOnNextLine: true
-PointerAlignment: Right
-SpaceBeforeParens: Always
-ColumnLimit: 100
-IndentWidth: 4
-AlignEscapedNewlines: DontAlign
-...
+Language:               Cpp
+Standard:               Cpp03
```

Comment:
There is a problem with SWIG version in RHEL. We need nested template parentheses to be separated by whitespace (`> >` instead of `>>`). On the other hand, Cpp03 doesn't support raw literals - it transforms `R"**` into `R "**` (-> compilation error).

Reply:
You can compile against devtoolset in RHEL base packages now, so don't worry about that.